RecoLocalCalo/HGCalRecAlgos/interface/HGCalRecHitSimpleAlgo.h

```diff
@@ -41,7 +41,8 @@ class HGCalRecHitSimpleAlgo : public HGCalRecHitAbsAlgo {
     //    float clockToNsConstant = 25;
     float energy = uncalibRH.amplitude() * adcToGeVConstant_;
     float time   = uncalibRH.jitter();
-    if(time<0) time   = 0; // fast-track digi conversion
+
+    //if(time<0) time   = 0; // fast-track digi conversion
```

Comment:
In this case time = -1 an allowable input, if the input rechit had its time set. 
There is a separate flag for knowing if the time is OK to use in the first place.